e2e/ublock.py

```diff
@@ -0,0 +1,30 @@
+from pathlib import Path
+import requests
+
+from autoparaselenium import Extension
+
+
+web_drivers = Path.home() / ".web-drivers"
+web_drivers.mkdir(parents=True, exist_ok=True)
+
+
+class installations:
```

Comment:
aren't you supposed to capitalize class names in python?

Reply:
I'm abusing classes to allow me to make variables with properties accessible via dot notation. I have essentially just made an object.

```python
installations.chrome.url
```

Reply:
buruh moment lmao ok